cmd/proxy/actions/sumdb.go

```diff
@@ -0,0 +1,37 @@
+package actions
+
+import (
+	"net/http"
+	"net/http/httputil"
+	"net/url"
+	"path"
+	"strings"
+)
+
+func sumdbPoxy(url *url.URL, nosumPatterns []string) http.Handler {
+	rp := httputil.NewSingleHostReverseProxy(url)
+	rp.Director = func(req *http.Request) {
+		req.Host = url.Host
+		req.URL.Scheme = url.Scheme
+		req.URL.Host = url.Host
+		req.URL.Path = strings.Replace(req.URL.Path, "/sumdb/"+url.Host, "", 1)
```

Comment:
Just for my understanding - we use `StripPrefix` above, do we use replace here so that the handler can be used on it's own?

Reply:
@marpio great catch 💯When I initially wrote the code, I didn't use `StripPrefix`, but then I realized adding it would be more robust and forgot to remove it here. I removed it and updated the PR. Let me know if it all looks good to you. Thanks!